src/dreyfus_index_manager.erl

```diff
@@ -86,6 +90,9 @@ handle_call({open_error, DbName, Sig, Error}, {OpenerPid, _}, State) ->
 
 handle_cast({cleanup, DbName}, State) ->
     clouseau_rpc:cleanup(DbName),
+    {noreply, State};
```

Comment:
Why do we add this here?

Reply:
@tonysun83  Are you referring to the line 93? If yes, we did not add this here, it was here before - this a standard way to finish `handle_cast`.  This line has changed as we added a new `handle_cast` clause.